pxd.c

```diff
@@ -1261,7 +1261,7 @@ static int pxd_init_disk(struct pxd_device *pxd_dev, struct pxd_add_ext_out *add
 	snprintf(disk->disk_name, sizeof(disk->disk_name),
 		 PXD_DEV"%llu", pxd_dev->dev_id);
 	disk->major = pxd_dev->major;
-	disk->minors = 256;
+	disk->minors = 1;
```

Comment:
No, this should not be set by us. It is initialized part of alloc_disk argument.

Reply:
Kernel 5.15 onwards this is required since alloc_disk doesn't do it. Will move it up.